packages/big-design/src/components/Datepicker/styled.ts

```diff
@@ -0,0 +1,143 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled from 'styled-components';
+
+import { ExtraProps } from './Datepicker';
+import { HeaderProps } from './Header';
+
+export const StyledHeader = styled.div<HeaderProps>`
+  align-items: center;
+  display: flex;
+  font-weight: ${({ theme }) => theme.typography.fontWeight.semiBold};
+  justify-content: space-between;
+`;
+
+export const StyledDatepicker = styled.div<ExtraProps>`
+  & .react-datepicker-wrapper {
+    display: inline-block;
+    width: 100%;
+  }
+
+  & .react-datepicker__input-container {
+    display: inline-block;
+    font-family: ${({ theme }) => theme.typography.fontFamily};
+    position: relative;
+    width: 100%;
+  }
+
+  & .react-datepicker__month-container {
+    margin: ${({ theme }) => theme.spacing.medium};
+  }
+
+  & .react-datepicker__month {
+    margin-top: ${({ theme }) => theme.spacing.xSmall};
+    text-align: center;
+  }
+
+  & .react-datepicker__header {
+    border-top-left-radius: ${({ theme }) => theme.spacing.xxSmall};
+    border-top-right-radius: ${({ theme }) => theme.spacing.xxSmall};
+    position: relative;
+    text-align: center;
+  }
+
+  & .react-datepicker__current-month {
+    font-weight: ${({ theme }) => theme.typography.fontWeight.semiBold};
+  }
+
+  & .react-datepicker__day-names,
+  .react-datepicker__week {
+    white-space: nowrap;
+  }
+
+  & .react-datepicker__day,
+  .react-datepicker__month-text {
+    cursor: pointer;
+  }
+
+  & .react-datepicker__day-name,
+  .react-datepicker__day {
+    color: ${({ theme }) => theme.colors.secondary60};
+    display: inline-block;
+    line-height: ${({ theme }) => theme.lineHeight.xLarge};
+    margin: ${({ theme }) => theme.spacing.xxSmall};
+    text-align: center;
+    width: ${({ theme }) => theme.lineHeight.xLarge};
+  }
+
+  & .react-datepicker__day:hover {
```

Comment:
The one state that we discussed offline, it's because when the calendar is opened the date is selected by default when we enable keyboard navigation. I might need to check this with design to get some resolution on this.